TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -25,18 +25,10 @@ TYPO3:
             label: 'Neos Backend'
             provider: 'PersistedUsernamePasswordProvider'
             requestPatterns:
-              'TYPO3.Neos:backendControllers':
-                pattern: 'ControllerObjectName'
+              'TYPO3.Neos:BackendControllers':
+                pattern: ControllerObjectName
                 patternOptions:
-                  controllerObjectNamePattern: 'TYPO3\Neos\Controller\.*'
-              'TYPO3.Neos:serviceControllers':
-                pattern: 'ControllerObjectName'
-                patternOptions:
-                  controllerObjectNamePattern: 'TYPO3\Neos\Service\.*'
-              'TYPO3.Neos:mediaControllers':
-                pattern: 'ControllerObjectName'
-                patternOptions:
-                  controllerObjectNamePattern: 'TYPO3\Media\Controller\.*'
+                  controllerObjectNamePattern: 'TYPO3\Neos\Controller\.*|TYPO3\Neos\Service\.*|TYPO3\Media\Controller\.*'
```

Comment:
this is a really bad regression reverting the newly formatted requestPatterns into the old format